core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/score/director/AbstractScoreDirector.java

```diff
@@ -385,7 +373,6 @@ public void afterEntityAdded(EntityDescriptor<Solution_> entityDescriptor, Objec
         if (lookUpEnabled) {
             lookUpManager.addWorkingObject(entity);
         }
-        variableListenerSupport.afterEntityAdded(entityDescriptor, entity);
```

Comment:
I don't understand. How can we get away with not calling `varibleListenerSupport`? Who calls it instead?